### DISCUSSION TITLE
Zooming (pinch to zoom equivalent) options

### DISCUSSION THREAD

**Discussion by limeafon** [noah-nuebling/mac-mouse-fix#260] [Questions] [answered]
Hi, I've been trying to figure a way to perform one-handed zooming with a mouse. Was wondering if there is a way to configure the existing cmd+scroll gesture to something like:

holding the middle button + scrolling, or 
holding the middle button + move mouse up/down?

**Comment by akira304moto** [comment]
Same request here. On old Logi mouse with Logi Options I was used to "Side button + scrolling" for single hand zooming.

**Comment by serjrudakov** [comment]
same here. It would be great to be able to hold a button and drag(right/left) to zoom in/out
I saw this option in Abnormal mouse https://abnormalmouse.intii.com, but they lack the custom action assignment. 
I love your app, if only it would have pith zoom emulation - it would be perfect for video editing workflow

**Comment by OptimusPrime91** [comment]
I found out you can cmd+scroll to zoom (although it's not mentioned anywhere). Are there other key combos?

**Comment by noah-nuebling** [comment, marked as answer]
Hey guys, marking this Discussion as answered since I'm moving the topic to Issue #360. Please continue discussion there. Thanks!

Oh and stay tuned for Mac Mouse Fix 3, I think you guys might really enjoy it :)
